src/OpenTelemetry/Metrics/MeterProviderSdk.cs

```diff
@@ -162,6 +162,16 @@ internal sealed class MeterProviderSdk : MeterProvider
                             try
                             {
                                 metricStreamConfig = viewConfig(instrument);
+
+                                if (metricStreamConfig is ExplicitBucketHistogramConfiguration
+                                    && instrument.GetType().GetGenericTypeDefinition() != typeof(Histogram<>))
+                                {
```

Comment:
metricStreamConfig = null; -- this statement got lost in new commit.

Reply:
🤦